src/gfp12.h

```diff
@@ -0,0 +1,29 @@
+#ifndef BN256_GFP12_H
+#define BN256_GFP12_H
+
+#include <gfp6.h>
+
+namespace bn256 {
+
+    // gfp12 implements the field of size p¹² as a quadratic extension of gfp6
+    // where ω²=τ.
+    struct gfp12 {
+        // value is xω + y
+        gfp6 x;
+        gfp6 y;
+
+        const gfp12& set(gfp12 &a) {
```

Comment:
Please be consistent with `gfp12& a` instead of `gfp12 &a` throughout the code.

Reply:
resolved with commit 185b5dc47da805da953d51ecf48b6943d71c9fe1